index.js

```diff
@@ -663,8 +659,6 @@ function buildArrayTypeCondition (type, accessor) {
           return buildArrayTypeCondition(subType, accessor)
         })
         condition = `(${conditions.join(' || ')})`
-      } else {
```

Comment:
this case is not possible because validation of types `schema is invalid: data/properties/ids/type must be equal to one of the allowed values` -> already covered by tests

Reply:
which line is already handling this already?

Reply:
https://github.com/fastify/fast-json-stringify/blob/c83cc32d59ff7814255e8c29523a86f9169e14cb/index.js#L32 whenever i try to set something for the type that is not in that `switch`

Reply:
concrete line https://github.com/fastify/fast-json-stringify/blob/c83cc32d59ff7814255e8c29523a86f9169e14cb/index.js#L88